Hackathon WIP (#211)
rapid7/insightconnect-plugins: rpm/help.md

##### Input

As input, the plugin takes the expected parts of a complete RPM package label: name<epoch:>-version-release.arch.

> For better accuracy in identifying the users intended package, the user is also asked to specify their distribution and architecture.

Reviewer: ```suggestion
For better accuracy in identifying the user's intended package, the user is also asked to specify their distribution and architecture.
```

Note that a label must match exactly to find a specific package, and label sections must be specified from left to right.
Thus, release info without the version is ignored when searching. A custom repository URL may also be specified, which will disable all other repos.

> If a custom key URL is also given (note: URL), then the key is used to do a check signature on the package downloaded before it is examined by RPM.

Reviewer: ```suggestion
If a custom key URL is also given, then the key is used to do a check signature on the package downloaded before it is examined by RPM.
```

Briefly, when given a package, the plugin checks to see if the cache already holds the results of am `rpm -qi --dump`.

> If it does not, yum downloader downloads the package and automatically performs a check signature before inspecting it and caching the result.

Reviewer: `yumdownloader` is a program https://linux.die.net/man/1/yumdownloader
```suggestion
If it does not, yumdownloader downloads the package and automatically performs a check signature before inspecting it and caching the result.
```

# Version History

> * 1.0.2 - Fix typos in yaml and help

Reviewer: We don't need this since this hasn't been released since last Hub changes. No version bump here.

Reviewer: @mhofert-r7 Still need to address this one.

Reviewer: We can remove this line
